app/assets/javascripts/admin/addon/controllers/admin-customize-colors-show.js

```diff
@@ -3,94 +3,96 @@ import I18n from "I18n";
 import bootbox from "bootbox";
 import discourseComputed from "discourse-common/utils/decorators";
 import { later } from "@ember/runloop";
+import { action } from "@ember/object";
+import copyText from "discourse/lib/copy-text";
```

Comment:
Oh this is odd, we have some duplication here. Lately I've improved and have changed some things to use `clipboardCopy` from `app/assets/javascripts/discourse/app/lib/utilities.js`. It does the same thing with the same fallback but also tries to use `writeText` first.

IMO we should get rid of `copyText` in favour of `clipboardCopy`

Reply:
Especially since it seems like `copyText` has not changed since 2015.

Reply:
Yes indeed we should get rid of this